src/connectors/recmusic.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const filter = new MetadataFilter({
+	all: katakanaHanToZen,
+});
+
+Connector.playerSelector = '.the-player';
+
+Connector.artistSelector = 'a.track-info-artist';
+
+Connector.albumSelector = 'a.track-info-album';
+
+Connector.getTrackArt = () => document.querySelector('.track-info-image img').style['background-image'].slice(5, -2);
```

Comment:
It's better to use `Util.extractUrlFromCssProperty` function to get a URL from CSS properties.